modules/ckanext-ytp_main/ckanext/ytp/templates/snippets/search_result_text.html

```diff
@@ -13,12 +13,18 @@
 
 #}
 
-{% if type == 'dataset' or type == 'showcase_dataset' or type == 'harvest' %}
+{% if (type == 'dataset' or type == 'showcase_dataset' or type == 'harvest') and request.path == '/dataset' %}
```

Comment:
Seems to work for me. I have done it this way, in order to separate the translation string of apisets and datasets and this could be done based on the request.path (whether the url has apiset in it or dataset)

Reply:
Fixed and tested locally, works for "harvest", "apiset", "dataset" and "showcase"

Reply:
Instead of hacky solution of looking at request path which might fail at some point if the same template is used from some other template, we should fix why the type is dataset and not apiset

Reply:
Fixed, so that the type for apiset is apiset not dataset